pypreprocess/subject_data.py

```diff
@@ -48,7 +48,15 @@
 #                  " it's OK. Also, the CoV typically spikes at the end and the "
 #                  "begining of the acquisition, due to saturation effects, "
 #                  "scanner instability, etc.")
-tsdiffana_tooltip = ("TODO")
+tsdiffana_tooltip = (
+    "(Squared) differences across sequential volumes. "
+    "A large value indicates an artifact "
+    "that occurred during the slice acquisition, possibly related to motion. "
+    " Note that such artifacts can be slice specific. "
+    " It may be a good option to remove an artefacted scan "
+    "from subsequent analyses. Large variations around the brain indicate "
+    " (uncorrected) motion effects. Large variations should be confined "
+    "to vascular structures or ventricles.")
```

Comment:
@bthirion  I'd have a list of 6 tooltip strings, 1 for each thumbnail you are showing (at the moment all the tsdiffana thumbs have the same tooltip) and then do something like this

```
    ...
    # create thumbnails
    thumbnails = []
    for output_filename, tooltip in zip(output_filenames, tooltips):
        thumbnail = Thumbnail(tooltip=tooltip)
        ...

```